### ISSUE TITLE
Review: 1.4.13 Content on Hover or Focus (AA)

### ISSUE THREAD

**Issue by qbalsdon** [GetEvinced/mcag#35]
Task: Edit [MCAG](https://github.com/GetEvinced/mcag/blob/main/src/app/data/mcag.json) section 1.4.13 considering the mobile research questions (to be completed)

---
Where receiving and then removing pointer hover or keyboard focus triggers additional content to become visible and then hidden, the following are true:

    Dismissible: A mechanism is available to dismiss the additional content without moving pointer hover or keyboard focus, unless the additional content communicates an input error or does not obscure or replace other content;

    Hoverable: If pointer hover can trigger the additional content, then the pointer can be moved over the additional content without the additional content disappearing;

    Persistent: The additional content remains visible until the hover or focus trigger is removed, the user dismisses it, or its information is no longer valid.

Exception: The visual presentation of the additional content is controlled by the user agent and is not modified by the author.
---
Sufficient Techniques for Success Criterion 1.4.13

Note: Other techniques may also be sufficient if they meet the success criterion. See [Understanding Techniques.](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140916/understanding-techniques.html#understanding-techniques)

    SCR39: Making content on focus or hover hoverable, dismissible, and persistent
    ARIA: Using role="tooltip" (Potential future technique)
    CSS: Using hover and focus pseudo classes (Potential future technique)

Failures for Success Criterion 1.4.13

    F95: Failure of Success Criterion 1.4.13 due to content shown on hover not being hoverable
    Failure to make content dismissable without moving pointer hover or keyboard focus (Potential future technique)
    Failure to meet by content on hover or focus not remaining visible until dismissed or invalid (Potential future technique)
